mkeditorpacks/README.md

```diff
@@ -0,0 +1,19 @@
+MkEditorPacks
+=============
+
+This a tool suite to build game packs for map editors.
+
+It is developped for the [Unvanquished](https://unvanquished.net/) project, but must fit any project supported by the editors listed below.
+
+Supported editors
+-----------------
+
+- [GtkRadiant](http://icculus.org/gtkradiant/)
+- [NetRadiant](https://gitlab.com/xonotic/netradiant)
+
+Authors
```

Comment:
Hey @neumond! Thanks you for that answer! :smiley: 
@Viech , which license do you prefer? for this kind of scripts a permissive one like BSD or ISC looks good IMO. I personally like the ISC one for its concision, but it looks like the BSD one is already used by Unvanquished/Daemon stuff, do what you prefer!